api/envoy/config/bootstrap/v2/bootstrap.proto

```diff
@@ -71,10 +71,10 @@ message Bootstrap {
 
     // [#not-implemented-hide:] Hide from docs.
     message DeprecatedV1 {
-      // This is the global :ref:`SDS <arch_overview_dynamic_config_sds>` config
+      // This is the global :ref:`SDS <arch_overview_dynamic_config_eds>` config
```

Comment:
@alyssawilk thoughts on whether to leave this for now or what to do with this given your other work?